terms.toml

```diff
@@ -875,6 +881,12 @@ term = "Strong typing"
 tags = []
 translation = "強型別"
 
+[[terms]]
+term = "Succinct non-interactive argument of knowledge"
+tags = []
+translation = "SNARK"
```

Comment:
Suggest changing the translation to 簡潔無交互知識證明

Reply:
I translated it to 無互動 instead of 交互 to make it more Taiwanese style. [49a3dc4](https://github.com/EtherTW/Terms/pull/22/commits/49a3dc49e62cdb3deda271077d502f275b679c92)